### PR TITLE
fix: pod count tooltip [15767]

### PR DESCRIPTION
**Pull request by smriti0710**
Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [x] The title of the PR conforms to the [Toolchain Guide](https://argo-cd.readthedocs.io/en/latest/developer-guide/toolchain-guide/#title-of-the-pr)
* [ ] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] Optional. My organization is added to USERS.md.
* [ ] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/blob/master/community/CONTRIBUTING.md#legal)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).
* [ ] My new feature complies with the [feature status](https://github.com/argoproj/argoproj/blob/master/community/feature-status.md) guidelines.
* [ ] I have added a brief description of why this PR is necessary and/or what this PR solves.

Issue:  https://github.com/argoproj/argo-cd/issues/15767 
Fixes [ISSUE 15767]"

https://github.com/argoproj/argo-cd/assets/29578254/131c5d06-8dce-4a6c-a779-0e0bf574883a

### CODE REVIEW

**Comment by smriti0710** [comment]
@crenshaw-dev Can you please review, also attached the video after the fix.

File: ui/src/app/applications/components/application-pod-view/pod-view.scss [hunk 1]
```diff
@@ -250,3 +250,7 @@ $pod-age-icon-clr: #ffce25;
 .tippy-content {
     text-align: left;
 }
+
+.tippy-popper {
+    z-index: 1 !important;
+}
```
Comment (crenshaw-dev) [inline ui/src/app/applications/components/application-pod-view/pod-view.scss:256]: I'm not up on SCSS... is this style scoped at all? Should we be worried about this style affecting tippy tooltips in other contexts?

Comment (smriti0710) [reply ui/src/app/applications/components/application-pod-view/pod-view.scss]: Used the tippy z-index prop to handle this.
doc for tippy.js : https://atomiks.github.io/tippyjs/v5/all-props/

**Comment by smriti0710** [comment]
addressed in https://github.com/argoproj/argo-cd/pull/15928

**Pull request closed without merging**